src/components/oInput.js

```diff
@@ -197,14 +235,23 @@ oInput.prototype.readOnly = function(readonly) {
 }
 
 oInput.prototype.required = function(required) {
-  if(typeof required !== 'boolean') return this;
+  if(required !== 'true' && required !== 'false' && typeof required !== 'boolean') return this;
 
   this.element.attr({
     required
   });
   return this;
 }
 
+oInput.prototype.step = function(step) {
+  if(step !== 'any' && isNaN(step)) return this;
+
+  this.element.attr({
+    step
+  });
+  return this;
+}
+
```

Comment:
Myślałem, żeby może tu dać tez możliwość wpisania w stringu liczby tak jak w metodach wyżej. By można było tylko dać jakieś Number(step) czy tam parseFloat(step). Ale to już jak uważasz, możesz nie zmieniać nic

Reply:
isNaN sprawdza i działa też na stringach.
![image](https://user-images.githubusercontent.com/34141138/132727246-73d8a94c-0ad7-4e31-98cc-cca5941615a6.png)